src/driver/communication_interface.cc

```diff
@@ -268,6 +269,13 @@ void CommunicationInterface::PublishLcmAndPauseStatus() {
     PublishRobotStatus();
     // TODO(@anyone): make pause status part of the robot status
     PublishPauseStatus();
+
+    // publish driver status
+    {
+      std::scoped_lock<std::mutex> lock {driver_status_mutex_};
```

Comment:
Prob move this to the PublishDriverStatus method; could also take a copy and get rid of the read lock entirely I think if that is desirable.